oauth2client/contrib/appengine.py

```diff
@@ -477,15 +469,15 @@ def _parse_state_value(state, user):
         user: google.appengine.api.users.User, The current user.
 
     Raises:
-        InvalidXsrfTokenError: if the XSRF token is invalid.
+        ValueError: if the XSRF token is invalid.
```

Comment:
Yep. In my opinion `_parse_state_value()` shouldn't be validating the state value, just returning it. Let the callback handler deal with the token validation, just like in `flask_util`: https://github.com/google/oauth2client/blob/master/oauth2client/contrib/flask_util.py#L410